http3/multipart.py

```diff
@@ -26,6 +26,10 @@ def render_headers(self) -> bytes:
         )
 
     def render_data(self) -> bytes:
+        if not isinstance(self.value, str):
+            if isinstance(self.value, bytes):
+                return self.value
+            self.value = str(self.value)
```

Comment:
I'm not sure I'm a big fan of this behavior? Why not raise a TypeError saying what we accept and forcing users to pass values that make sense to this interface?

Reply:
replied in issue post #119